physical_property/README.md

```diff
@@ -28,11 +29,12 @@ Effective permeability (log *P*<sub>app</sub>) was measured by Parallel Artifici
 
 ## What's here
 
-- `SAMPL7_molecule_ID_and_SMILES.csv` - CSV file containing SAMPL7 challenge molecule IDs and SMILES provided by the Ballatore lab. Note, compounds SM09 and SM10 were experimental controls and will not be analyzed in the challenge, but they are still provided here.
-- `log P`: Folder will be updated soon to contain challenge input files, instructions and a submission template.
-- `pKa`: Folder will be updated soon to contain challenge input files, instructions and a submission template.
-- `permeability`: Folder will be updated soon to contain challenge input files, instructions and a submission template.
-- `images`: Folder containing images of the molecules for this challenge in PDF and JPEG format.
+- [`SAMPL7_molecule_ID_and_SMILES.csv`](SAMPL7_molecule_ID_and_SMILES.csv): CSV file containing SAMPL7 challenge molecule IDs and SMILES. SMILES were provided by the Ballatore lab.
```

Comment:
Does this have the "as provided" IDs or those after renumbering?

(If those after renumbering, where are you saving the key for sorting these back out later?)

Reply:
> Does this have the "as provided" IDs or those after renumbering?
The IDs in that file are the renumbered IDs. 

> If those after renumbering, where are you saving the key for sorting these back out later?
I made a an [experimental_data/](https://github.com/samplchallenges/SAMPL7/tree/physical_property/physical_property/experimental_data) 
folder that has a file that matches the experimental molecule IDs to the renumbered SAMPL7 challenge molecule IDs

Reply:
Perfect, thanks.